src/constants/blockchains.js

```diff
@@ -69,3 +68,13 @@ export const BLOCKCHAINS = {
     }
   }
 };
+
+function isTestChain (chain) {
```

Comment:
I am not certain this should be in the constants, but I cannot think of a better place immediately